x/concentrated-liquidity/store_test.go

```diff
@@ -184,3 +205,113 @@ func (s *KeeperTestSuite) TestParseFullPositionFromBytes() {
 		})
 	}
 }
+
+func (s *KeeperTestSuite) TestParseFullTickFromBytes() {
+	const (
+		emptyKeySeparator   = ""
+		invalidKeySeparator = "-"
+	)
+
+	var (
+		cdc = s.App.AppCodec()
+
+		formatFullKey = func(tickPrefix []byte, poolIdBytes []byte, tickIndexBytes []byte) []byte {
+			key := make([]byte, 0)
+			key = append(key, tickPrefix...)
+			key = append(key, poolIdBytes...)
+			key = append(key, tickIndexBytes...)
+			return key
+		}
```

Comment:
I would prefer not to move it outside of the test because there are helpers in `types` that allow to format a position's key correctly such as [`types.KeyTick`](https://github.com/osmosis-labs/osmosis/blob/cb7fc937c4bd9262fa3d637d139f9cf94b8c1565/x/concentrated-liquidity/types/keys.go#L73-L78)

The reason why we have this `formatFullKey` helper here is that I want to be testing injecting the wrong prefix. There should not be a need for that outside of these tests